packages/eslint/core.js

```diff
@@ -60,8 +60,8 @@ module.exports = {
     ],
 
     // who doesn't love a good nested ternary?
-    'no-nested-ternary': 'off',
-    'unicorn/no-nested-ternary': 'off',
+    'no-nested-ternary': 'warn',
+    'unicorn/no-nested-ternary': 'warn',
```

Comment:
I'm more of a type-a personality. Should we go "all the way" with `error` instead?

Reply:
nah, after sleeping on it, it seems better as a "suggestion" / warning like you did here